x/ccv/provider/types/params.go

```diff
@@ -97,8 +99,6 @@ func DefaultParams() Params {
 			clienttypes.Height{}, // latest(initial) height
 			commitmenttypes.GetSDKSpecs(),
 			[]string{"upgrade", "upgradedIBCState"},
-			true,
```

Comment:
Do we need migration for this? The `TemplateClient` needs to be changed.

Reply:
No, the underlying structure is the same (`07-tendermint` client).

This is just a function API change (it takes less params).

v7:
- https://github.com/cosmos/ibc-go/blob/7fb65c76000f207ece1b83d8950e3aaff3dfc0e7/modules/light-clients/07-tendermint/tendermint.pb.go#L38

v4:
- https://github.com/cosmos/ibc-go/blob/9b3d7ef7e7fdd9f100a76553955fca7600883aeb/modules/light-clients/07-tendermint/types/tendermint.pb.go#L38